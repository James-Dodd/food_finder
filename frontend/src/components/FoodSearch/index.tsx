import React from "react"
import styled from "styled-components"

const StyledH1 = styled.h1`
  color: red;
  font-size: 80px;
`

export default function FoodSearch() {
  return (
    <form>
      <label>
        Search For Food:
        <input type="text" name="food" id="food" />
      </label>
      <input type="submit" value="Submit" />
    </form>
  )
}
