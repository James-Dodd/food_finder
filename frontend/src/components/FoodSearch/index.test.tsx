import React from "react"
import { shallow, mount } from "enzyme"
import FoodSearch from "./index"

describe("Header", () => {
  it("renders shallow", () => {
    const component = shallow(<FoodSearch />)
    expect(component).toMatchSnapshot()
  }),
    it("renders mount", () => {
      const component = mount(<FoodSearch />)
      expect(component).toMatchSnapshot()
    })
})
