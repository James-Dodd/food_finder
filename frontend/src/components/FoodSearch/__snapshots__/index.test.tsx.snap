// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders mount 1`] = `
<FoodSearch>
  <div>
    <form
      onSubmit={[Function]}
    >
      <label>
        Search For Food:
        <input
          id="food"
          name="food"
          onChange={[Function]}
          type="text"
          value=""
        />
      </label>
      <input
        type="submit"
        value="Submit"
      />
    </form>
    <div>
      <h2>
        Searched foods
      </h2>
    </div>
  </div>
</FoodSearch>
`;

exports[`Header renders shallow 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <label>
      Search For Food:
      <input
        id="food"
        name="food"
        onChange={[Function]}
        type="text"
        value=""
      />
    </label>
    <input
      type="submit"
      value="Submit"
    />
  </form>
  <div>
    <h2>
      Searched foods
    </h2>
  </div>
</div>
`;
